<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SynergiesGUI</class>
 <widget class="QMainWindow" name="SynergiesGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Experiments Synergies</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Arm_Length</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Max_Force</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Force_%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Experiment Data Input&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="connectButton_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton:checked {

background-color: rgb(0, 170, 0);

}</string>
     </property>
     <property name="text">
      <string>SAVE</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>401</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Target Controller&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="VR_Button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>VR Target</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="UR5_Button">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>UR5</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="define_Button">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Define</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>381</width>
       <height>221</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QToolButton" name="VRprevButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Prev</string>
       </property>
      </widget>
      <widget class="QToolButton" name="VRNextButton">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>100</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
      <widget class="QToolButton" name="VRHomeButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>81</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QToolButton" name="getPosButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Get Pos</string>
       </property>
      </widget>
      <widget class="QLabel" name="displayPos">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Position Display</string>
       </property>
      </widget>
      <widget class="QLabel" name="displayHome">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>HOME POSITION</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QToolButton" name="genCoorButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>351</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Generate Target Coordinates</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QToolButton" name="URPrevButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Prev</string>
       </property>
      </widget>
      <widget class="QToolButton" name="URNextButton">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>100</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
      <widget class="QToolButton" name="URHomeButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>81</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="record_Button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton:checked {

background-color: rgb(200, 18, 2);

}</string>
     </property>
     <property name="text">
      <string>RECORD</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="stop_Button_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="trialNumber">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton:checked {

background-color: rgb(0, 170, 0);

}</string>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalinga</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DATE</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>311</width>
       <height>141</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Experiment_Notes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="connectButton_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton:checked {

background-color: rgb(0, 170, 0);

}</string>
    </property>
    <property name="text">
     <string>Select DIR</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Python_console">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>571</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Python Console Output</string>
    </property>
   </widget>
   <widget class="QToolButton" name="connectButton_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton:checked {

background-color: rgb(0, 170, 0);

}</string>
    </property>
    <property name="text">
     <string>Load Config</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="connectButton_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>650</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton:checked {

background-color: rgb(0, 170, 0);

}</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionLicense"/>
    <addaction name="actionContact"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionContact">
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>VR_Button</tabstop>
  <tabstop>UR5_Button</tabstop>
  <tabstop>define_Button</tabstop>
  <tabstop>VRprevButton</tabstop>
  <tabstop>VRNextButton</tabstop>
  <tabstop>VRHomeButton</tabstop>
  <tabstop>getPosButton</tabstop>
  <tabstop>genCoorButton</tabstop>
  <tabstop>URPrevButton</tabstop>
  <tabstop>URNextButton</tabstop>
  <tabstop>URHomeButton</tabstop>
  <tabstop>record_Button</tabstop>
  <tabstop>stop_Button_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
